css/reset-css.md

```diff
@@ -0,0 +1,142 @@
+# Reset.css
+
+### Projected Time
+
+About 30 minutes
+- Lesson: 4.5 min
```

Comment:
😂

Reply:
Feel free to round to the minute. Actually, i really think the whole lesson will take longer than 4 minutes. Include time for the materials section too.

Reply:
😂